src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -26,195 +27,44 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.energy.IEnergyStorage;
 
 import javax.annotation.Nullable;
 import java.util.List;
 
-public class TestPower extends TileEntityInventoryBase implements ITeslaProducer, net.minecraft.util.ITickable, IWailaBodyMessage, ITeslaConsumer, ITeslaHolder {
-    private BaseTeslaContainer container = new BaseTeslaContainer(0, 10000, 1000, 1000);
-    private InternalInventory inventory = new InternalInventory(this, 1);
-    private boolean[] flag = {false, false, false, false, false, false};
-    private int T0transfer = 100;
+public class TestPower extends TileEntityInventoryBase implements net.minecraft.util.ITickable ,IWailaBodyMessage {
+    private BaseTeslaContainer container = new BaseTeslaContainer(0,10000,1000,1000);
+    private InternalInventory inventory = new InternalInventory(this, 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "T0transfer" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00116)